src/graph.cpp

```diff
@@ -229,4 +236,61 @@ std::vector<Vertex *> Graph::findEndOfLines(const int stationId) const {
     }
 
     return eol_stations;
-}
\ No newline at end of file
+}
+
+/**
+ * DFS traversal variation that sets the visited attribute to true of the nodes the DFS traverses to
+ * Time Complexity: O(|V|+|E|)
+ * @param source - Vertex where the DFS starts
+ */
+void Graph::visitedDFS(Vertex *source) {
+    source->setVisited(true);
+    for (Edge *e : source->getAdj()){
+        if (!e->getDest()->isVisited()){
+            visitedDFS(e->getDest());
+        }
+    }
+}
+
+/**
+ * Finds the pairs of stations with max Max-Flow
+ * Time Complexity: O(V^2*(VE^2)) //TODO someone check if V^2 is the correct ammount of times EdmondsKarp is called
```

Comment:
I think the complexity of this, as you had, should be V³E², because: first cycles runs V times, second cycle runs at worst V cycles too, and inside them, in the worst case that a single vertex is connected to all others, a VE² edmonds-karp is run for all of them

Reply:
Yeah, its was just temporary notation so the reasoning behind it was more explicit
Will simplify now